dev/ci/scripts/ci-elpi_test.sh

```diff
@@ -9,7 +9,6 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/elpi"
   export DUNE_build_FLAGS="--release"
-  make test-core
-  make examples
-  make test-apps
+  make -j1 all-tests
+  make -j1 all-examples
```

Comment:
What's going on here?

Reply:
is there some reason not to do `dune build --root . @runtest`?

Reply:
make is calling dune multiple times in parallel and dune doesn't like it (I got error messages like "dune is already running, if you're sure, remove _build/.lock")

Reply:
It wouldn't be just @runtest, there are a few diferent targets.

Reply:
C.f. https://github.com/LPCIC/coq-elpi/pull/785